filters/apiusagemonitoring/filter_test.go

```diff
@@ -297,27 +313,84 @@ func Test_Filter_AllHttpMethodsAreSupported(t *testing.T) {
 				"https://www.example.org/lapin/malin",
 				200,
 				func(t *testing.T, pass int, m *metricstest.MockMetrics) {
-
-					httpCountMetricKey := fmt.Sprintf(
-						"apiUsageMonitoring.custom.<unknown>.<unknown>.%s.<unknown>.http_count",
-						testCase.expectedMethodInMetric)
-					httpStatusClassCountMetricKey := fmt.Sprintf(
-						"apiUsageMonitoring.custom.<unknown>.<unknown>.%s.<unknown>.http2xx_count",
+					pre := fmt.Sprintf(
+						"apiUsageMonitoring.custom.<unknown>.<unknown>.%s.<unknown>.",
 						testCase.expectedMethodInMetric)
-
 					assert.Equal(t,
 						map[string]int64{
-							httpCountMetricKey:            int64(pass),
-							httpStatusClassCountMetricKey: int64(pass),
+							pre + "http_count":    int64(pass),
+							pre + "http2xx_count": int64(pass),
 						},
 						m.Counters,
 					)
+					assert.Contains(t, m.Measures, pre+"latency")
+				})
+		})
+	}
+}
 
-					latencyMetricKey := fmt.Sprintf(
-						"apiUsageMonitoring.custom.<unknown>.<unknown>.%s.<unknown>.latency",
-						testCase.expectedMethodInMetric)
+func Test_Filter_PathTemplateMatchesInternalSlashes(t *testing.T) {
+	testWithFilter(
+		t,
+		createFilterForTest,
+		http.MethodPost,
+		"https://www.example.org/foo/orders/1/2/3/order-items/123",
+		204,
+		func(t *testing.T, pass int, m *metricstest.MockMetrics) {
+			pre := "apiUsageMonitoring.custom.my_app.my_api.POST.foo/orders/:order-id/order-items/:order-item-id."
+			assert.Equal(t,
+				map[string]int64{
+					pre + "http_count":    int64(pass),
+					pre + "http2xx_count": int64(pass),
+				},
+				m.Counters,
+			)
+			assert.Contains(t, m.Measures, pre+"latency")
+		})
+}
 
-					assert.Contains(t, m.Measures, latencyMetricKey)
+func Test_Filter_PathTemplateMatchesInternalSlashesTooFollowingVarPart(t *testing.T) {
+	filterCreate := func() (filters.Filter, error) {
+		args = []interface{}{`{
+				"application_id": "my_app",
+				"api_id": "my_api",
+			  	"path_templates": [
+                  "foo/:a",
```

Comment:
Please fix indentation.

Reply:
@tkrop : Fixed in 3d5fb1c79c454f8349d1241d06d7a57d1e38c939